samples/scripts/PowerShell/GenerateOpenApiYaml.ps1

```diff
@@ -0,0 +1,33 @@
+<#
+.SYNOPSIS 
+Generates the OpenApi doc for the specified version and compares it with the baseline to make sure no breaking changes are introduced
+.Parameter WorkingDir
+The working directory
+.PARAMETER OutputPathofDll
+Path for the web projects dll
+.PARAMETER OutputPathOfOpenApiDoc
+Path to output the OpenApi Doc (in yaml)
+.PARAMETER Versions
+Api versions to generate the OpenApiDoc for and compare with baseline
+#>
+
+param(
+    [string]$WorkingDir,
```

Comment:
Do we need `$WorkingDir` or can the `$OutputPathofDll` always be absolute?

Reply:
Changed to be swagger path to use to get swagger and output files